code/game/machinery/doors/windowdoor.dm

```diff
@@ -215,6 +227,16 @@
 	health = 300.0 //Stronger doors for prison (regular window door health is 150)
 
 
+/obj/machinery/door/window/brigdoor/allowed(mob/M)
+	if(inoperable()) // Brigdoors are the exception to the "fail open" windoor - they lock closed
+		return FALSE
+	. = ..()
```

Comment:
Perhaps this should generate a message of some kind to tell people why it doesn't open?

Reply:
Yeah, not a bad idea.